Translation/Translation/Transliteration.cs

```diff
@@ -0,0 +1,105 @@
+﻿
+namespace Translation
+{
+    public class Transliteration
+    {
+        const string RUSSIAN = "абвгдеёжзийклмнопрстуфхцчшщъыьэюя";
+        const string LATIN = " abvgdeyozhiklmnprstufhc";
+        public Transliteration() { }
+
+        // This method checks the string for validity
```

Comment:
You have two separate methods for check string and translate. What about you call Translate not checking string? Think better check string in setter, and than translate